### DISCUSSION TITLE
rAudio-1 User Guide

### DISCUSSION THREAD

**Discussion by apennc** [rern/rAudio#1033] [General]
Recently found rAudio-1 in my quest for Pi 4 with DAC hat software. My main goal is multi room/zone playback. I've tried out Moode, Volumio and PiCore Player but they each have had issues with either some of my audio files (m4a), cover art images or not mounting my NAS share point. So far rAudio has had none of these issues but I have run into another issue.

The user guide is not displaying any of the pages. I see the headings and the page counter but pages are not displaying at all. All I see is a very small icon of some sort. I thought it might be because I'm running a PiHole DNS server but after changing to Google DNS servers & taking my PiHole server off line, nothing changed. Is there somewhere I can just download the guide as a pdf?

Thanks

**Comment by rern** [comment]
It's a bug which will be fixed by next update.
In the meantime:
```sh
dir=/srv/http/assets/img/guide
mkdir -p $dir
find /srv/http/assets/img -maxdepth 1 -type f -name '[0-9]*' -exec mv {} $dir \;
```
